docs/ingestion-beam/sink-job.md

```diff
@@ -179,7 +179,7 @@ Note: `-Dexec.args` does not handle newlines gracefully, but bash will remove
 ### Locally
 
 If you install Java and maven, you can invoke `mvn` in the following commands
-instead of using `./bin/mvn`; be aware, though, that Java 8 is the target JVM
+instead of using `./bin/mvn`; be aware, though, that Java 11 is the target JVM
```

Comment:
I expect now we'll see these warnings even when using `./bin/mvn` since we're using Java 11 everywhere.

Reply:
I haven't run into warnings since we started using `maven.compiler.release`, and if we were then CI should fail because we set `<compilerArgument>-Werror</compilerArgument>`